minimum-constraints.txt

```diff
@@ -392,8 +397,8 @@ Send2Trash==1.8.0
 sh==1.12.14
 simplegeneric==0.8.1
 singledispatch==3.7.0; python_version == '2.7'
-smmap==3.0.5; python_version == '2.7'
-smmap==5.0.0; python_version >= '3.5'
+# KS: issue with smmap 5.0.0 requiring gitdb >= 4 and gitdb 4.0.1
```

Comment:
Just out of curiosity: smmap 5.0.0 on python_version >= '3.5' has worked before. How did it fail now?

Reply:
DONE in my commit on this PR.

Reply:
Agree with your changes except that there is no min definition for gitdb with python 3.5 or 3.6 and there is a definition for gitdb2 for these python versions. Actually the relation between gitdb and gitdb2 is not even python version based (see their issue https://github.com/gitpython-developers/gitdb/issues/59) So I suggest we simply state that gitdb2 is a mirror for gitdb and acts as the minimum for certain python versions) an leave it at that.

Reply:
I made that change to simply say that # gitdb2 is a mirror Pypi name for certain gitdb versions.